### ISSUE TITLE
can you tell me more details about your program?

### ISSUE THREAD

**Issue by nanshou555** [zhangjinyangnwpu/HSI_Classification#5]
i am a fresh man to this area,can you tell me the enviroment such as the version of tensorflow,i tested your program on pycharm(windows),but there exists  errors.i wish you can give me some advices ,thank you very much

**Comment by zhangjinyangnwpu** [comment]
python version is 3.6 and tensorflow version is 1.8，in deep learning based method, you can change the profile with parser in main file and just run main; in traditional machine learning based methods, change configuration in top of the file，I think that may easy to read.
in deeplearning based methods, the data_loader.py define the dataset class, which can generate dateset with tfrecords format and read it, in model.py, I define the whole model which contain the network structure, loss function and how to train and test. Alse there has a get_decode_map function to get the final classification map. The traditional methods I just use the package of sklearn, hope 
 can help, sorry for late replay.
